script.js

```diff
@@ -16,4 +16,140 @@ for (let i = 0; i < mobileMenuItems.length; i += 1) {
 const closeMenu = document.querySelector('#toggle-x');
 closeMenu.addEventListener('click', () => {
   menu.classList.add('display-none');
-});
\ No newline at end of file
+});
+
+const projectsArr = [
```

Comment:
Please add the `X` sign to the pop detail page for mobile. Kindly see the screenshot below for reference.
![cross](https://user-images.githubusercontent.com/35504149/136089054-eb46e276-9277-4cc8-9611-c4f0046b348e.png)